src/main/frontend/worker/db_listener.cljs

```diff
@@ -30,12 +31,29 @@
         (shared-service/broadcast-to-clients! :sync-db-changes data))
 
       (when-not from-disk?
+        ;; Remove old :logseq.property.embedding/hnsw-label-updated-at when importing a graph
+        (when (and
+               (d/entity @conn :logseq.property.embedding/hnsw-label-updated-at)
+               (some (fn [d] (and (= (:a d) :db/ident) (= (:v d) :logseq.kv/import-type))) (:tx-data tx-report'))) ; imported graph
+          (let [tx-data (->> (d/datoms @conn :avet :logseq.property.embedding/hnsw-label-updated-at)
+                             (map (fn [d]
+                                    [:db/retract (:e d) :logseq.property.embedding/hnsw-label-updated-at])))]
+            (d/transact! conn tx-data {:skip-refresh? true})))
```

Comment:
we shouldn't have multiple d/transact! in sync-db-to-main-thread, and it may break the logic of 'broadcast-to-clients'?
we should do transact in invoke-hooks